webcompat/static/css/development/components/comment.css

```diff
@@ -84,6 +84,15 @@
     background : #fff;
   }
 
+    /* modifier new comment of existing bug */
+    .wc-Comment-wrapper-new {
```

Comment:
@magsout thanks for the review. I think either way with --new or --comment the purpose of the style will be a little ambiguous just coming from the css. I feel new is slightly more descriptive but can replace it with comment.

I assume the editor will probably know the style's use by inspection from the webpage.